lib/SPE.js

```diff
@@ -524,7 +524,7 @@ SPE.ShaderAttribute.prototype.flagUpdate = function() {
 	'use strict';
 
 	var attr = this.bufferAttribute,
-		range = attr.updateRange;
+		range = attr.addUpdateRange;
```

Comment:
I don't understand this change, addUpdateRange is a function, not an object.
https://threejs.org/docs/#api/en/core/BufferAttribute.addUpdateRange
https://threejs.org/docs/#api/en/core/BufferAttribute.updateRanges

Reply:
Good catch!  I blindly followed the deprecation message that said updateRange had been replaced by addUpdateRange.

Reply:
in r164, this is the warning 
https://github.com/mrdoob/three.js/blob/c6d4b83c11e573662347f017d6a057b431179b89/src/core/BufferAttribute.js#L50
updateRange is a getter returning the _updateRange object.
You need to change indeed the code to use `attr.addUpdateRange(start, count)` (start instead of offset) if I understand it.